app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Nice work adding the `:confirmable` option to Devise :clap: However Devise is throwing `NameError` whenever a user try to register (see image :point_down:). And the error seems to be because you forgot to add the `confirmable` migration. Kindly create a new migration file to add the confirmable's columns to the users tables.
![Screenshot from 2023-12-13 13-19-13](https://github.com/nilab-khwaja/blog-app/assets/91142432/3f445017-979e-4287-8c82-8885671d9470)